spec/safe_statements_spec.rb

```diff
@@ -1723,10 +1815,10 @@ def up
 
             expect do
               test_migration.suppress_messages { test_migration.migrate(:up) }
-            end.to raise_error(PgHaMigrations::InvalidMigrationError, "Table \"foos3\" is not a partitioned table")
+            end.to raise_error(PgHaMigrations::InvalidMigrationError, /Partitioned table "foos3" contains sub-partitions/)
           end
 
-          it "raises error when sub-partitioning detected" do
+          it "raises error when sub-partition indexes have not been created and if_not_exists is true" do
```

Comment:
Is this also `recurse` is false?

Reply:
Yup, I'll update the description

Reply:
If we're missing sub partition indexes and `recurse = false` then won't we have an error regardless of the status of `if_not_exists`?

Reply:
Yes, it's just that the error messaging is slightly different